server/__init__.py

```diff
@@ -182,6 +182,14 @@ def create_app():
     if app.config['LOCAL']:
         os.environ['OAUTHLIB_INSECURE_TRANSPORT'] = '1'
 
+    if app.config['API_PROJECT']:
+        secret_client = secretmanager.SecretManagerServiceClient()
+        secret_name = secret_client.secret_version_path(cfg.API_PROJECT,
```

Comment:
@Fructokinase This is another resource that has to be added to the mixer project.